Lunch/LunchPFTableViewController.swift

```diff
@@ -0,0 +1,138 @@
+//
+//  LunchPFTableViewController.swift
+//  Lunch
+//
+//  Created by Spencer Amann on 8/13/15.
+//  Copyright (c) 2015 Spencer Amann. All rights reserved.
+//
+
+import ParseUI
+import Parse
+import UIKit
+
+class LunchPFTableViewController: PFQueryTableViewController {
+
+    let classKey = "Eateries"
+    let placeKey = "place"
+    let voteKey = "vote"
+
+    var newEateryName: String?
+
+    override init(style: UITableViewStyle, className: String?) {
+        super.init(style: style, className: className)
+    }
+
+    required init!(coder aDecoder: NSCoder!) {
+        super.init(coder: aDecoder)
+
+        self.parseClassName = "Eateries"
+        self.pullToRefreshEnabled = true
+        self.paginationEnabled = false
+    }
+
+    override func viewDidLoad() {
+        self.navigationItem.hidesBackButton = true
+    }
+
+    override func queryForTable() -> PFQuery {
+        let query = PFQuery(className: classKey)
+
+        query.orderByDescending(voteKey)
+        return query
+    }
+
+    override func tableView(tableView: UITableView, cellForRowAtIndexPath indexPath: NSIndexPath, object: PFObject?) -> PFTableViewCell? {
+        var cell = tableView.dequeueReusableCellWithIdentifier("lunchPlaceCell") as! PFTableViewCell!
+        if cell == nil {
```

Comment:
`cell` will never be `nil` because you're force-unwrapping and it would have crashed before reaching this line. I think you mean to use `as?` and I don't know why you have `PFTableViewCell!` instead of `PFTableViewCell`